src/ontology/mondo-edit.obo

```diff
@@ -44203,7 +44203,6 @@ synonym: "neoplasm of hepatobiliary system" EXACT [MONDO:patterns/neoplasm]
 synonym: "tumor of hepatobiliary system" EXACT [MONDO:patterns/neoplasm]
 synonym: "tumour of hepatobiliary system" EXACT OMO:0003005 []
 xref: DOID:0080355 {source="MONDO:equivalentTo"}
-xref: DOID:3117 {source="MONDO:mondoIsBroaderThanSource"}
```

Comment:
I think this statement is correct since [DOID:3117](https://www.ebi.ac.uk/ols4/ontologies/doid/classes?obo_id=DOID%3A3117) is 'hepatobiliary benign neoplasm', however until the Synonym Sync pipeline is updated to not add DOID:3117 as a source for the exact synonym 'hepatobiliary tumors' I do not know how else to get the qc checks to pass other than remove this for now.